MobilityData/mobility-feed-api
# chore: set user agent to the integration tests requests

**Summary:**

With this PR the integration tests send a custom user agent with all requests. The user agent format is: 
`MobilityData Feed API Integration Tests (Python {platform.python_version()})`
Example:
`MobilityData Feed API Integration Tests (Python 3.11.4)`

Part of MobilityData/product-tasks#88.
Out of scope: Filtering the API usage per user-agent, this will be done after production deployment.

**Expected behavior:** 

The custom user agent is sent with the integration tests requests

**Testing tips:**

- Checkout the branch
- Download the latest sources.csv from https://bit.ly/catalogs-csv
- Login and grab a refresh token from https://qa.mobilitydatabase.org
- Execute the following command:
```
REFRESH_TOKEN={YOUR_REFRESH_TOKEN} && ./scripts/integration-tests.sh -u https://api-qa.mobilitydatabase.org -f {YOUR_SOURCES_CSV_FULL_FILENAME_PATH}
```
- Go to GCP console and verify that the proper user-agent is recorded in the logs. Not all log records contains the user-agent, to locate records that contains user-agent information, filter by _logName="projects/mobility-feeds-qa/logs/run.googleapis.com%2Frequests"_

Please make sure these boxes are checked before submitting your pull request - thanks!
